docs/installation.rst

```diff
@@ -67,6 +67,29 @@ support this::
 
     pip install scales
 
+*Optional:* Column-Level Encryption (CLE) Support
+--------------------------------------------------
+The driver has built-in support for client-side encryption and
+decryption of data. Please see :doc:`column_encryption` for more detail
+about this feature, including how to configure and use it.  This
+feature relies on the `cryptography <https://cryptography.io/en/latest/>`_ module
+for its implementation. The initial release of the CLE feature in Python driver 3.27.0
+explicitly listed cryptography as a required dependency; as a result the module will be
+automatically downloaded and installed when that version of the driver is installed.
+Later versions of the driver do not specify this module as a requirement, so if you
+want to use the CLE feature with a version of the driver after 3.27.0 you will need
+to take additional steps to install the cryptography module.
+
+You can install this module along with the driver by specifying the `cle` extra::
+
+    pip install cassandra-driver[cle]
+
+Alternatively, you can also install the module directly via `pip`::
+
+    pip install cryptography
+
+Any version of cryptography >= 35.0 will work for the CLE feature.  You can find additional
+details at `PYTHON-1351 <https://datastax-oss.atlassian.net/browse/PYTHON-1351>`_
```

Comment:
I meant to include the link to the relevant JIRA ticket in my initial version and just forgot it.  We don't have any other links to PYTHON tickets in this doc but it seemed like a good idea for this case which (as discussed above) is a bit more complicated than a normal doc change.